src/Components/Footer/Footer.scss

```
@@ -0,0 +1,94 @@
+.footer {
+  background-color: #252525;
+  color: white;
+  font-size: 60px;
+
+  .gridContainer {
+    display: grid;
+    grid-template-rows: 1fr 1fr;
+    grid-template-columns: 2fr 1fr 1fr 1fr;
```

```suggestion
    grid-template-rows: repeat(2, 1fr);
    grid-template-columns: 2fr repeat(3, 1fr);
```
이런식으로도 사용할 수 있습니다, 수정은 안하셔도 됩니다!